techmatters/flex-plugins
# Summary tab - Ability to add contact to case

Primary reviewer: @stephenhand 

## Description

- New feature: Modifying the existing “Add to New Case” button on the Summary tab of an Active Contact.

### Checklist

### Related Issues
Fixes CHI-1972

### Verification steps

- Create an offline contact 
-  Verify that we have a new "Add to New Case” drop-up button with "New Case" and "Existing Case" as options 
- The "New Case” button should behave the same as the previous “Add Contact to Case” button
- The “Existing Case” button should take you to the Search tab
